roles/build-release/tasks/main.yaml

```
@@ -0,0 +1,59 @@
+---
+- name: Install antsibull from current source in a virtualenv with poetry
+  command: poetry install
+  args:
+    chdir: "{{ playbook_dir | dirname }}"
+
+- name: Ensure the sdist directory exists
+  ansible.builtin.file:
+    name: "{{ antsibull_sdist_dir }}"
+    state: directory
+    mode: 0755
+
+- name: Clone the ansible-build-data git repository
+  ansible.builtin.git:
+    repo: "{{ antsibull_data_git_repo }}"
+    dest: "{{ antsibull_data_git_dir }}"
+    version: "{{ antsibull_data_version }}"
+    # TODO: Building a release means we may create or change files which would be overwritten by force
+    force: yes
+
+- name: Set expected path to deps file and release archive
+  ansible.builtin.set_fact:
+    _deps_file: "ansible-{{ antsibull_ansible_version }}.deps"
+    _release_archive: "{{ antsibull_sdist_dir }}/ansible-{{ antsibull_ansible_version }}.tar.gz"
+
+- name: Build a release with new dependencies
+  command: >
+    poetry run antsibull-build single {{ antsibull_ansible_version }} \
+      --data-dir {{ antsibull_data_dir }} \
+      --sdist-dir {{ antsibull_sdist_dir }} \
```

I'm pretty sure escaping EOL is unnecessary
```suggestion
  command: >-
    poetry run antsibull-build single {{ antsibull_ansible_version }}
      --data-dir {{ antsibull_data_dir }}
      --sdist-dir {{ antsibull_sdist_dir }}
```

---

@webknjaz yup, thanks -- old habit, will fix :)